osu.Game.Rulesets.Osu/Objects/SliderTailCircle.cs

```diff
@@ -29,7 +29,7 @@ public class LegacyTailJudgement : OsuJudgement
 
         public class TailJudgement : SliderEndJudgement
         {
-            public override HitResult MaxResult => HitResult.LargeTickHit;
+            public override HitResult MaxResult => HitResult.SliderTailHit;
```

Comment:
Just to confirm, this is 100% intended to only affect non-classic mod scores?

Reply:
Yeah, this would be the intention. `LargeTickHit` matches classic scoring closer.

Reply:
I think @peppy is misunderstanding. Classic mod is using `SmallTickHit`, and the reason it is so is because the slider judgment (the proportional one) is the one that applies the combo. Classic mod is still not quite 1:1.

So yes, it is intended that this only affects non-classic-mod scores.